data-prepper-plugins/opensearch-source/build.gradle

```diff
@@ -0,0 +1,28 @@
+plugins {
+    id 'java'
+}
+
+group 'org.opensearch.dataprepper'
+version '2.3.0-SNAPSHOT'
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    implementation project(path: ':data-prepper-api')
+    implementation 'com.fasterxml.jackson.dataformat:jackson-dataformat-csv'
```

Comment:
I see a lot of dependencies here which do not seem to be needed. Please remove the dependencies which are not in use.

Reply:
Thanks for you review. Changes are incorporated.